jenkins-jobs/pipelines/release/release-charts-pipeline.groovy

```diff
@@ -0,0 +1,189 @@
+
+RELEASE_SEM_VERSION=RELEASE_VERSION.replaceAll(/\.(?=[^.]*$)/, '-').toLowerCase()
+
+MAVEN_CENTRAL = 'https://repo1.maven.org/maven2'
+
+DEBEZIUM_OPERATOR_DIR='operator'
+DEBEZIUM_PLATFORM_DIR='platform'
+
+
+node('release-node') {
+    catchError {
+        stage('Validate parameters') {
+            common.validateVersionFormat(RELEASE_VERSION)
+        }
+
+        stage('Initialize') {
+            dir('.') {
+                deleteDir()
+                sh "git config user.email || git config --global user.email \"debezium@gmail.com\" && git config --global user.name \"Debezium Builder\""
+                sh "ssh-keyscan github.com >> $HOME_DIR/.ssh/known_hosts"
+            }
+            dir(GPG_DIR) {
+                withCredentials([
+                        string(credentialsId: 'debezium-ci-gpg-passphrase', variable: 'PASSPHRASE'),
+                        [$class: 'FileBinding', credentialsId: 'debezium-ci-secret-key', variable: 'SECRET_KEY_FILE']]) {
+                    echo 'Creating GPG directory'
+                    def gpglog = sh(script: "gpg --import --batch --passphrase $PASSPHRASE --homedir . $SECRET_KEY_FILE", returnStdout: true).trim()
+                    echo gpglog
+                }
+            }
+            checkout([$class                           : 'GitSCM',
+                      branches                         : [[name: "*/$DEBEZIUM_OPERATOR_BRANCH"]],
+                      doGenerateSubmoduleConfigurations: false,
+                      extensions                       : [[$class: 'RelativeTargetDirectory', relativeTargetDir: DEBEZIUM_OPERATOR_DIR], [$class: 'CloneOption', noTags: false, depth: 1]],
+                      submoduleCfg                     : [],
+                      userRemoteConfigs                : [[url: "https://$DEBEZIUM_OPERATOR_REPOSITORY", credentialsId: GIT_CREDENTIALS_ID]]
+            ]
+            )
+
+            checkout([$class                           : 'GitSCM',
+                      branches                         : [[name: "*/$DEBEZIUM_PLATFORM_BRANCH"]],
+                      doGenerateSubmoduleConfigurations: false,
+                      extensions                       : [[$class: 'RelativeTargetDirectory', relativeTargetDir: DEBEZIUM_PLATFORM_DIR], [$class: 'CloneOption', noTags: false, depth: 1]],
+                      submoduleCfg                     : [],
+                      userRemoteConfigs                : [[url: "https://$DEBEZIUM_PLATFORM_REPOSITORY", credentialsId: GIT_CREDENTIALS_ID]]
+            ]
+            )
+        }
+
+        stage("Install helm") {
+            sh 'curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3'
+            sh 'chmod 700 get_helm.sh'
+            sh './get_helm.sh'
+            sh 'helm version'
+        }
+
+        stage("Install GitHub CLI") {
+            sh 'curl -fLjsO https://github.com/cli/cli/releases/download/v2.67.0/gh_2.67.0_linux_amd64.tar.gz'
```

Comment:
Given there is a version hardcoded it should probably be extracted and defined as a constant.

Reply:
This is mostly related to https://github.com/debezium/debezium/pull/6183. I'll do the changes on that branch

Reply:
Addressed on the relative PR.